README.rst

```diff
@@ -355,43 +355,38 @@ To ensure parity with the real redis, there are a set of integration tests
 that mirror the unittests.  For every unittest that is written, the same
 test is run against a real redis instance using a real redis-py client
 instance.  In order to run these tests you must have a redis server running
-on localhost, port 6379 (the default settings).  The integration tests use
-db=10 in order to minimize collisions with an existing redis instance.
+on localhost, port 6379 (the default settings). **WARNING**: the tests will
+completely wipe your database!
```

Comment:
Are you still using `db=10`? I guess a warning is in order in any case.

Reply:
No, it hasn't used db=10 for a long time - and some tests use multiple databases.